src/views/BoardEvents.vue

```diff
@@ -11,10 +11,10 @@
       ref="eventCreateDialog"
       :subjects="subjects"
       :initial-date="new Date().toISOString().split('T')[0]"
-      :value="$route.name === 'BoardCreateEventAll' || $route.name === 'BoardEditEventAll'"
-      :edit="$route.name === 'BoardEditEventAll' || lastDialogState.edit"
+      :value="$route.name === 'BoardEditEventAll'"
+      :edit="true"
```

Comment:
This can be simplified further
```suggestion
      edit
```